Examples/Scripts/FreeSurferPipelineBatch-long.sh

```diff
@@ -109,18 +109,14 @@ for i in ${!Subjlist[@]}; do
     --subject="$Subject" \
     --path="$StudyFolder" \
     --sessions="$TPlist" \
-    --template-id="$LongitudinalTemplate" \
-    --extra-reconall-arg-long=-T2pial \
-    --extra-reconall-arg-base=-T2pial \
-    --extra-reconall-arg-base=-T2 \
-    --extra-reconall-arg-base=$StudyFolder/${Timepoints[0]}/T1w/T2w_acpc_dc_restore.nii.gz)
+    --template-id="$LongitudinalTemplate")
 
   #--extra-reconall-arg-base=-conf2hires Freesurfer reports this is unneeded.
   echo "Running command: ${cmd[*]}"
-  "$cmd[@]}"
+  $cmd[@]
```

Comment:
Huh?  That doesn't work, please change it back.
```
tim@timsdev:~$ foo=(bar baz)
tim@timsdev:~$ echo $foo[@]
bar[@]
tim@timsdev:~$ echo "${foo[@]}"
bar baz
```
The purpose of this syntax is to be able to handle whitespace (when quoted or escaped properly in the input arguments), for instance in folder names:
```
tim@timsdev:~$ function countargs() { echo $#; }
tim@timsdev:~$ args=('1 2' '3 4')
tim@timsdev:~$ countargs ${args[@]}
4
tim@timsdev:~$ countargs "${args[@]}"
2
```

Reply:
that was fixed in later commit

Reply:
fixed in previous commit